stdlib/LinearAlgebra/src/LinearAlgebra.jl

```diff
@@ -385,7 +385,7 @@ control over the factorization of `A`.
     these are already in a factorized form
 
 # Examples
-```jldoctest
+```julia-repl
```

Comment:
I wonder if it might be better to retain the doctests, and change the final line to
```julia
julia> Y ≈ A\X
true
```
This is what the inplace operation should evaluate to, so it makes the intent of the last line clear. The exact values obtained as the solution are not really relevant.

Reply:
> and change the final line to

I pushed a commit to restore these two doctests, but I assumed you meant to replace the last _two_ lines, because both `Y` _and_ `A\X` are different depending on the OpenBLAS kernels.

Reply:
Yes, thanks, that's what I meant